public/app/features/alerting/unified/mockGrafanaNotifiers.ts

```diff
@@ -289,7 +289,7 @@ export const grafanaAlertNotifiersMock: NotifierDTO[] = [
         label: 'Message',
         description:
           'Optional message. You can use templates to customize this field. Using a custom message will replace the default message',
-        placeholder: '',
+        placeholder: '{{ template "default.message" . }}',
```

Comment:
The notifiers endpoint returns `{{ template ...` for everything that is supposed to be a template. We use this to heuristically work out if we show the template picker, so I had to update this mock so I could test the email settings logic